@mindflayer was quick to submit a fix at https://github.com/mindflayer/python-mocket/pull/192. Thank you so much! Indeed, `pytest --no-cov test/test_lorawan.py` now works with Mocket 3.10.9 - excellent!

However, in the context of this PR, or when _all tests_ are being invoked (just run `pytest`), we are observing:
```python
Traceback (most recent call last):
  File "/path/to/terkin-datalogger/.venv310/lib/python3.10/site-packages/mocket/mocket.py", line 349, in true_sendall
    encoded_response = hexload(response_dict["response"])
KeyError: 'response'
```
-- https://github.com/hiveeyes/terkin-datalogger/actions/runs/3610611746/jobs/6084514008#step:7:3427

Because @mindflayer told us about our unconventional usage of Mocket already, we should probably go back to the drawing board.

--- a/requirements-test.txt
+++ b/requirements-test.txt
@@ -8,7 +8,11 @@ pyfakefs<6
 esp32-machine-emulator<2
 fake-rpi<1
 
-mocket>=3.10.9,<3.11
+# Use a modern version of Mocket in general, but downgrade on Python 3.11.
+# https://github.com/mindflayer/python-mocket/pull/181
+mocket>=3.10.9,<3.11; python_version<"3.11"
+mocket<3.9; python_version>="3.11"
+
 httpretty<1
 pytest-httpserver<2
 

